Package.swift

```diff
@@ -48,6 +48,7 @@ let package = Package(
     .library(name: "SwiftRefactor", type: .static, targets: ["SwiftRefactor"]),
   ],
   targets: [
+    .target(name: "_InstructionCounter"),
```

Comment:
I'm a bit curious on why those files are in a new target and not just in the CLI target?

Reply:
Because SwiftPM does not support mixed-language targets, so you can’t have Swift and C in the same target.